Resources/Prototypes/_RMC14/Entities/Clothing/Uniforms/marines.yml

```
@@ -312,10 +312,19 @@
     unfoldVerbText: rmc-sleeves-verb-unfold
     foldVerbText: rmc-sleeves-verb-fold
 
+- type: entity
+  parent: RMCJumpsuitASOFormal
+  id: RMCJumpsuitASOFormalAlt
+  name: auxiliary support officer formal uniform
```

```suggestion
  name: auxiliary support officer formal uniform
  suffix: Alternative
```
You wouldn't be able to tell in the spawn menu between these 2 uniforms by name

---

done